txn_writer.go

```diff
@@ -464,8 +464,6 @@ func diffLocks(l1 gps.Lock, l2 gps.Lock) *LockDiff {
 				diff.Add = append(diff.Add, add)
 				i2next = i2 + 1 // Don't evaluate to this again
 				continue        // Keep looking for a matching project
-			case +1: // Project has been removed, handled below
-				break
```

Comment:
I think it probably makes it a bit more visually complete for the casual reader not necessarily familiar with `strings.Compare()` to go with the empty branch alternative.